autorest/src/main/java/com/microsoft/embeddedsocial/autorest/EmbeddedSocialBatchedClientImpl.java

```diff
@@ -125,13 +134,13 @@ private void processBatchResponse(Response batchResponse) throws IOException  {
     }
 
     // Caller must call issue batch to send the batch to ES. This code is not thread-safe,
-    // and is assumed that is called exactly once.
-    public void issueBatch() throws InterruptedException, IOException {
+    // and is assumed that is called exactly once. Returns the HTTP response of the batch call.
+    public Response issueBatch() throws InterruptedException, IOException {
 
-        // Check if the issueBatch called aready
+        // Check if the issueBatch called already
         synchronized (this.issueBatchAlreadyCalled) {
             if (this.issueBatchAlreadyCalled) {
-                return;
+                throw new IllegalStateException("Batch Already Issued");
```

Comment:
Throughout this file, all our error messages are inconsistent. Sometimes we use upper-case, sometimes lower-case. The rest of the autorest library uses upper-case for the first word of the error message, and lower-case for the rest. We should do the same thing throughout this code. Thanks. 😀